### ISSUE TITLE
Fetching Deposits inquiry

### ISSUE THREAD

**Issue by thanhlai** [Intacct/intacct-sdk-net#164]
Hello there,

Could you please give us a few pointers on how to retrieve AR/Deposits (and/or Deposits Manual) records via APIs or the SDK?
Thank you.
![Screen Shot 2021-06-07 at 9 24 41 AM](https://user-images.githubusercontent.com/6494781/121055541-338cd980-c772-11eb-9863-95bfd5d09c04.png)

**Comment by thanhlai** [comment]
Just to clarify it a bit more: what **object** do i use to retrieve that list?

**Comment by dylan-holt-sage** [comment]
Hey @thanhlai,

The object to use for retrieving the information for **Deposits** are the Deposits API or Deposits object for SDK which are located within  the **Cash Management** directory. The records for Deposits in Cash Management and AR should be the same. 
I have verified this on my end to be the case :)

Here is a screenshot taken from the API Reference:

![image](https://user-images.githubusercontent.com/66645799/125654872-6811c4ee-b0e3-40e7-8aa6-4b1739d10d92.png)

Within the reference should contain examples to help you with your retrieval, here's a quick snippet of what Deposit allows:

![image](https://user-images.githubusercontent.com/66645799/125655287-2f5d1ab8-8391-4d35-82f5-c8c7150df215.png)


Closing this inquiry.
